cluster-autoscaler/core/scaledown/planner/planner_test.go

```diff
@@ -346,6 +463,37 @@ func generateReplicaSets() []*appsv1.ReplicaSet {
 	}
 }
 
+func generateReplicaSetWithReplicas(name string, specReplicas, statusReplicas int32, labels map[string]string) []*appsv1.ReplicaSet {
+	return []*appsv1.ReplicaSet{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: "default",
+				UID:       rSetUID(name),
+			},
+			Spec: appsv1.ReplicaSetSpec{
+				Replicas: &specReplicas,
+				Selector: metav1.SetAsLabelSelector(labels),
+			},
+			Status: appsv1.ReplicaSetStatus{
+				Replicas: statusReplicas,
+			},
+		},
+	}
+}
+
+func rSetUID(name string) types.UID {
+	return types.UID(fmt.Sprintf("api/v1/namespaces/default/replicasets/%s", name))
+}
+
+func scheduledPodWithRSOwner(name string, cpu, memory int64, nodeName string, rsUID types.UID, labels map[string]string) *apiv1.Pod {
```

Comment:
nit: Both `rsUID` and `labels` are always set to the same value in all tests. Why not just extend `scheduledPod` below to use these constants?

Reply:
Done.